.circleci/config.yml

```diff
@@ -110,24 +158,21 @@ jobs:
                 when: always
 
 
-    "python-3.6":
+    "python-3.6-v0":
         docker:
             - image: circleci/python:3.6-stretch-browsers
 
         environment:
             PERCY_ENABLED: True
-            PERCY_PROJECT: plotly/dash-table-python
+            PERCY_PROJECT: plotly/dash-table-python-v0
 
         steps:
             - checkout
 
             - run:
                 name: Inject Percy Environment variables
                 command: |
-                  echo 'export PERCY_TOKEN="$PERCY_PYTHON_TOKEN"' >> $BASH_ENV
-
-            - restore_cache:
-                key: deps1-{{ .Branch }}-{{ checksum "requirements.txt" }}
+                  echo 'export PERCY_TOKEN="$PERCY_PYTHON_TOKEN_V0"' >> $BASH_ENV
```

Comment:
v0 and v1 visual tests end up in https://percy.io/plotly/dash-table-python-v0 and https://percy.io/plotly/dash-table-python-v1